src/GeoDataFrames.jl

```diff
@@ -4,15 +4,19 @@ import ArchGDAL as AG
 using DataFrames
 using Tables
 import GeoFormatTypes as GFT
-import GeoInterface
+import GeoInterface as GI
 using DataAPI
 using Reexport
 
 @reexport using Extents
 @reexport using GeoFormatTypes
 
 include("exports.jl")
+include("sources.jl")
```

Comment:
This could be `drivers.jl` be use you suffix with `XXDriver` ? Or `backends? I want to change this from sources in Rasters anyway